packages/contracts/src/utils/web3_wrapper.ts

```diff
@@ -5,15 +5,33 @@ import { Provider } from 'ethereum-types';
 
 import { coverage } from './coverage';
 
-export const txDefaults = {
+const testProvider = process.env.TEST_PROVIDER || 'ganache';
```

Comment:
Can we please make the test providers into an enum. I'd prefer we don't repeat string literal values in order to ease future refactoring.

Reply:
👍